test/spec/modules/dailymotionBidAdapter_spec.js

```diff
@@ -170,9 +172,8 @@ describe('dailymotionBidAdapterTests', () => {
     expect(reqData.coppa).to.be.true;
     expect(reqData.request.auctionId).to.eql(bidRequestData[0].auctionId);
     expect(reqData.request.bidId).to.eql(bidRequestData[0].bidId);
-    expect(reqData.request.mediaTypes.video.api).to.eql(bidRequestData[0].mediaTypes.video.api);
-    expect(reqData.request.mediaTypes.video.playerSize).to.eql(bidRequestData[0].mediaTypes.video.playerSize);
-    expect(reqData.request.mediaTypes.video.startdelay).to.eql(bidRequestData[0].mediaTypes.video.startdelay);
+
+    expect(reqData.request.mediaTypes.video).to.eql(bidRequestData[0].mediaTypes.video);
```

Comment:
isn't it better to have the unit test covering each attribute than the full object? then when one attribute is failing we will have the exact attribute where an issue is occuring instead of the whole object?